fe/fe-core/src/test/java/com/starrocks/catalog/IcebergTableTest.java

```diff
@@ -49,6 +50,7 @@ public class IcebergTableTest {
 
     @Before
     public void setUp() {
+        catalogName = "catalog0";
```

Comment:
this table for test is created manually. there is no need to test it. catalog must be empty

Reply:
removed